.github/workflows/cd.yml

```diff
@@ -20,13 +20,11 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           cache: "pnpm"
-      - name: Set publishing config
-        run: pnpm config set '//registry.npmjs.org/:_authToken' "${NPM_AUTH_TOKEN}"
-        env:
-          NPM_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
       - name: Install dependencies
         run: pnpm recursive install
       - name: Build packages
         run: pnpm build:packages
       - name: Publish all packages
-        run: pnpm publish:packages
+        run: pnpm config set '//registry.npmjs.org/:_authToken' "${NODE_AUTH_TOKEN}" && pnpm publish:packages
```

Comment:
Is this working now @mnemitz?

Reply:
Yeah I tested by triggering the action on this branch and it finally worked